optout/src/main/java/gov/cms/ab2d/optout/OptOutHandler.java

```diff
@@ -31,13 +35,21 @@ public void handleRequest(InputStream inputStream, OutputStream outputStream, Co
         //ToDo: replace on S3
         ClassLoader classLoader = getClass().getClassLoader();
         try {
-            File file = new File(Objects.requireNonNull(classLoader.getResource("optOutDummy.txt")).getFile());
+            File file = new File(Objects.requireNonNull(classLoader.getResource(Constants.fileName)).getFile());
+            //for s3
```

Comment:
I really like the use of a Constants file as a form of configuration for things like this. Well done!